### PR TITLE
fix: Text in reflection cards is not selectable during vote period

### PR DESCRIPTION
**Pull request by 87prashant**
# Description

fixes: #6310

## Final checklist

- [x] I checked the [code review guidelines](../docs/codeReview.md)
- [x] I have added [Metrics Representative](../docs/codeReview.md#metrics-representative) as reviewer(s) if my PR invovles metrics/data/analytics related changes
- [x] I have performed a self-review of my code, the same way I'd do it for any other team member
- [x] I have tested all cases I listed in the testing scenarios and I haven't found any issues or regressions
- [x] Whenever I took a non-obvious choice I added a comment explaining why I did it this way
- [ ] I added the label `One Review Required` if the PR introduces only minor changes, does not contain any architectural changes or does not introduce any new patterns and I think one review is sufficient'
- [x] PR title is human readable and could be used in changelog

### CODE REVIEW

**Comment by 87prashant** [comment]
Btw, I don't know why the import files showing changed!

**Comment by Dschoordsch** [comment]
> Btw, I don't know why the import files showing changed!

We have a pre-commit hook which also sorts imports. If you make changes to a file which was written before this hook, it will re-order the imports.

@JimmyLv can you please take a look?

File: packages/client/components/ReflectionCard/ReflectionCard.tsx [hunk 1]
```diff
@@ -216,7 +216,7 @@ const ReflectionCard = (props: Props) => {
     phases,
     isSpotlightSource
   )
-  const userSelect = readOnly ? (phaseType === 'discuss' ? 'text' : 'none') : undefined
+  const userSelect = readOnly ? (phaseType === 'discuss' || 'vote' ? 'text' : 'none') : undefined
```
Comment (JimmyLv) [inline packages/client/components/ReflectionCard/ReflectionCard.tsx:219]: -1 I didn't quite get the logic here, when the `phaseType` is `vote`, the result of `'vote' === 'discuss'` is FALSE and then the default value is taken as `'vote'`? This shouldn't be what we want.

**Review by JimmyLv — CHANGES_REQUESTED** [review]
If you want to allow the `vote` phase to be able to select text, you should be making a condition like this?
```js
if (phaseType === 'discuss' || phaseType === 'vote') {
  ...
}
```

File: packages/client/components/ReflectionCard/ReflectionCard.tsx (hunk 1 above)
Comment (87prashant) [reply packages/client/components/ReflectionCard/ReflectionCard.tsx:219]: Thank you @JimmyLv for clarifying the difference between the two:
My code made reflection cards selectable in every phase.

**Review by JimmyLv — APPROVED** [review]
LGTM!

**Pull request merged**